xarchain/assetlist.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "xarchain",
+  "assets": [
+    {
+      "description": "stXAR is the native cryptocurrency of the Arcana app chain, designed to orchestrate bridge-less interoperability between multiple blockchains",
+      "extended_description": "stXAR powers the Arcana app chain by supporting network consensus through staking, ensuring security and trustless coordination across chains.",
+      "denom_units": [
+        {
+          "denom": "stxar",
+          "exponent": 18
+        }
+      ],
+      "base": "stxar",
+      "name": "Xarchain staking token",
+      "display": "stxar",
+      "symbol": "STXAR",
```

Comment:
on CoinGecko, the symbol is just XAR, not STXAR??

Reply:
Can we have separate name on coin gecko and cosmos stake token ?